base/statefulset.yaml

```diff
@@ -89,12 +89,26 @@ spec:
             - "-ecx"
             # The use of qualified `hostname -f` is crucial:
             # Other nodes aren't able to look up the unqualified hostname.
-            - "exec /cockroach/cockroach start --logtostderr=WARNING --certs-dir /cockroach/cockroach-certs --listen-addr=:26357 --sql-addr=:26257 --advertise-addr $(hostname -f) --http-addr 0.0.0.0 --join cockroachdb-0.cockroachdb:26257,cockroachdb-0.cockroachdb:26357,cockroachdb-1.cockroachdb:26257,cockroachdb-1.cockroachdb:26357,cockroachdb-2.cockroachdb:26257,cockroachdb-2.cockroachdb:26357 --cache 25% --max-sql-memory 25%"
+            - >
+              exec /cockroach/cockroach start
+              --logtostderr=WARNING
+              --certs-dir /cockroach/cockroach-certs
+              --listen-addr=:26357
+              --sql-addr=:26257
+              --advertise-addr $(hostname -f)
+              --http-addr 0.0.0.0
+              --join cockroachdb-0.cockroachdb:26257,cockroachdb-0.cockroachdb:26357,cockroachdb-1.cockroachdb:26257,cockroachdb-1.cockroachdb:26357,cockroachdb-2.cockroachdb:26257,cockroachdb-2.cockroachdb:26357
+              --cache $(CACHE)
+              --max-sql-memory $(MAX_SQL_MEMORY)
```

Comment:
Should these env vars be documented somewhere (in the README?)?

Reply:
Yes? I don't know how much context you want to add though, like "these values are good with X requests/limits" and where it starts to step on the toes of the info in "documentation" repo.

Reply:
> Yes? I don't know how much context you want to add though, like "these values are good with X requests/limits" and where it starts to step on the toes of the info in "documentation" repo.

I was just thinking to document their existence: e.g."The value of `--max-sql-memory` can be set by the `MAX_SQL_MEMORY` environment variable"

Reply:
https://github.com/utilitywarehouse/cockroachdb-manifests/pull/81